src/app/routes/api/v3/admin/forms/__tests__/admin-forms.routes.spec.ts

```diff
@@ -265,4 +265,179 @@ describe('admin-form.routes', () => {
       })
     })
   })
+
+  describe('DELETE /forms/:formId/logic/:logicId', () => {
+    it('should return 200 with success message on successful form logic delete for email mode form', async () => {
+      // Arrange
+      const formLogicId = '606d408f37af650047dd1a6a'
```

Comment:
should we use `new ObjectId()` instead of hardcoding a string?

Reply:
edited